src/copyright/agent/database.cc

```diff
@@ -288,6 +289,7 @@ std::vector<unsigned long> CopyrightDatabaseHandler::queryFileIdsForUpload(int a
 #ifdef IDENTITY_COPYRIGHT
     , agentId, agentId
 #endif
+    , IDENTITY
     , agentId
```

Comment:
Please put `, IDENTITY, agentId` in an `#else` block to match the input format string.